lib/pbench/agent/tool_meister.py

```diff
@@ -73,11 +71,13 @@
 )
 from pbench.agent.toolmetadata import ToolMetadata
 from pbench.agent.utils import collect_local_info
+from pbench.common.logger import HostnameFilter
+from pbench.common.utils import md5sum
 
 
 # Logging format string for unit tests
 fmtstr_ut = "%(levelname)s %(name)s %(funcName)s -- %(message)s"
-fmtstr = "%(asctime)s %(levelname)s %(process)s %(thread)s %(name)s %(funcName)s %(lineno)d -- %(message)s"
+fmtstr = "%(asctime)s %(levelname)s %(hostname)s %(process)s %(thread)s %(name)s %(funcName)s %(lineno)d -- %(message)s"
```

Comment:
Why do we need to add the host name when we have it here (https://github.com/distributed-system-analysis/pbench/blob/main/lib/pbench/agent/redis.py#L212)?

Reply:
This was originally part of some experimentation to cover the gaps before we get the `RedisHandler` installed. What I probably really need is a log listener thread on `stderr` pipes for the `ssh` commands... which could be reported locally with the proper annotation, so maybe the filter wouldn't have been necessary anyway.

In any case, when I decided to scale back and just do the log level handoff, I didn't remove this. And you're right that as things are it doesn't help; some stuff remains stranded locally where the context is implicit, and once we get the `RedisHandler` it annotates the records.